orbit/pkg/keystore/keystore_darwin.go

```diff
@@ -0,0 +1,158 @@
+//go:build darwin && cgo
+
+package keystore
+
+/*
+   #cgo LDFLAGS: -framework CoreFoundation -framework Security
+   #include <CoreFoundation/CoreFoundation.h>
+   #include <Security/Security.h>
+*/
+import "C"
+import (
+	"errors"
+	"fmt"
+	"strings"
+	"unsafe"
+)
+
+const service = "com.fleetdm.fleetd.enroll.secret"
```

Comment:
Double checking: Orbit runs as `com.fleetdm.orbit` any issues with having a different sub-domain?

Reply:
This is just the name that shows up in the keychain. We just have to make sure any future items we put in a keychain has a unique name.